### PR TITLE
pkg: update some samsung packages

### PR DESCRIPTION
**Pull request by prakharb5**

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -3535,7 +3535,7 @@
   },
   "com.knox.vpn.proxyhandler": {
     "list": "Oem",
-    "description": "Third-party package that provides caller profile information to help consumers identify incoming calls and block unwanted ones.\nhttps://en.wikipedia.org/wiki/Hiya_(company)\nhttps://hiya.com/\nNOTE: Never trust a company that promotes spam blocking features.\nhttps://itmunch.com/robocall-caught-sending-customers-confidential-data-without-consent/\nHave a look at their privacy policy. It's... pretty scary: https://hiya.com/fr/hiya-data-policy\nNeeded for Samsung Smart Call (com.samsung.android.smartcallprovider)\nSamsung Knox allows business and personal content to \"securely\" coexist on the same handset.\nThis package handles proxies alongside KNOX.",
+    "description": "Samsung Knox allows business and personal content to \"securely\" coexist on the same handset.\nThis package handles proxies alongside KNOX.",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:3538]: Any reason why the Wikipedia pages have been removed?

File: resources/assets/uad_lists.json [hunk 2]
```diff
@@ -5090,7 +5090,7 @@
   },
   "com.samsung.ucs.agent.boot": {
     "list": "Oem",
-    "description": "Boot agent\nUCS is a company which has partnered with Samsung to provide licenses for Samsung Knox\nhttps://www.ucssolutions.com/blog/samsung-knox/\nI don't have precise information about the package itself but there are chances that it verifies some files on boot. If theses files are not verified\nthen it prevent the phone to boot \n",
+    "description": "Boot agent\nUCS is a company which has partnered with Samsung to provide licenses for Samsung Knox\nhttps://www.ucssolutions.com/blog/samsung-knox/\nI don't have precise information about the package itself but there are chances that it verifies some files on boot. If these files are not verified then it prevent the phone to boot.\n",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:5093]:
```suggestion
    "description": "Boot agent\nUCS is a company which has partnered with Samsung to provide licenses for Samsung Knox\nhttps://www.ucssolutions.com/blog/samsung-knox/\nI don't have precise information about the package itself but there are chances that it verifies some files on boot. If these files are not verified then it prevents the phone to boot.\n",
```

Comment (AnonymousWP) [reply resources/assets/uad_lists.json:5093]: Uhm, also, if it is like that, shouldn't it be `Unsafe` then? It's now `Recommended`.

File: resources/assets/uad_lists.json [hunk 3]
```diff
@@ -5170,7 +5170,7 @@
   },
   "com.sec.android.app.bluetoothtest": {
     "list": "Oem",
-    "description": "Bluetooth Test\nHidden feature accessible by entering *#*#232331#*#* in the samsung dialer\nWas calling home back in 2015 (https://forum.xda-developers.com/galaxy-s5/help/bluetoothtest-apk-calling-home-t3035182)\n",
+    "description": "Bluetooth Test\nHidden feature accessible by entering *#*#232331#*#* in the Samsung dialer.\nIt waas found to be calling home back in 2015 (https://forum.xda-developers.com/galaxy-s5/help/bluetoothtest-apk-calling-home-t3035182)\n",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:5173]:
```suggestion
    "description": "Bluetooth Test\nHidden feature accessible by entering *#*#232331#*#* in the Samsung dialer.\nIt was found to be calling home back in 2015:\nhttps://forum.xda-developers.com/galaxy-s5/help/bluetoothtest-apk-calling-home-t3035182",
```

File: resources/assets/uad_lists.json [hunk 4]
```diff
@@ -6084,7 +6084,7 @@
   },
   "com.samsung.android.app.smartcapture": {
     "list": "Oem",
-    "description": "Lets you group widgets on OneUI 4",
+    "description": "Samsung Capture\nSamsung's implementation of screenshot. Show the bottom toolbar after taking a screenshot\n\nLets you group widgets on OneUI 4 (?)",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:6087]:
```suggestion
    "description": "Samsung Capture\nSamsung's implementation of screenshot. Show the bottom toolbar after taking a screenshot\nLets you group widgets on OneUI 4 (?)",
```

File: resources/assets/uad_lists.json [hunk 5]
```diff
@@ -6124,7 +6124,7 @@
...
+    "description": "Safe to remove if not using VoWifi.\n#\nHuh... that's hard to explain quickly. Let's try : \nFirst you need to know that there is a standards organization which develops protocols for mobile telephony called 3GPP (mobile network : 2G/3G/4G/5G).\nRAT (physical connection method to a wireless communication network) non specified by the 3GPP is called non-3GPP. RAT Wifi is non-3GPP.\n\nePDG means Evolved Packet Data Gateway. The main role of the ePDG is to provide security mechanisms such as IPsec tunnelling of connections with the device over an untrusted non-3GPP access. It is mainly use for VoWifi (Voice over wifi).\nFor further information : \n- https://www.3gpp.org/technologies/keywords-acronyms/100-the-evolved-packet-core\n- https://www.aptilo.com/solutions/mobile-data-offloading/3gpp-wifi-access/\n- https://en.wikipedia.org/wiki/System_Architecture_Evolution#Evolved_Packet_Core_(EPC)\n",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:6127]:
```suggestion
    "description": "Safe to remove if you're not using VoWifi.\nFirst you need to know that there is a standards organization which develops protocols for mobile telephony called 3GPP (mobile network : 2G/3G/4G/5G).\nRAT (physical connection method to a wireless communication network) non specified by the 3GPP is called non-3GPP. RAT Wifi is non-3GPP.\nePDG means Evolved Packet Data Gateway. The main role of the ePDG is to provide security mechanisms such as IPsec tunnelling of connections with the device over an untrusted non-3GPP access. It is mainly used for VoWifi (Voice over wifi). For more information:\n- https://www.3gpp.org/technologies/keywords-acronyms/100-the-evolved-packet-core\n- https://www.aptilo.com/solutions/mobile-data-offloading/3gpp-wifi-access/\n- https://en.wikipedia.org/wiki/System_Architecture_Evolution#Evolved_Packet_Core_(EPC)",
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

File: resources/assets/uad_lists.json (hunk 1 above)
Comment (prakharb5) [reply resources/assets/uad_lists.json:3538]: That description was related to another app, not this one.

File: resources/assets/uad_lists.json (hunk 2 above)
Comment (prakharb5) [reply resources/assets/uad_lists.json:5093]: I didn't see that. Will make the change.

File: resources/assets/uad_lists.json (hunk 4 above)
Comment (prakharb5) [reply resources/assets/uad_lists.json:6087]: I added the extra \n since those points are different. I'd say keeping that extra space is better

Comment (AnonymousWP) [reply resources/assets/uad_lists.json:6087]: That's not necessary and we'd like to keep it consistent across packages in the list (meaning it's not the case for other packages as well).

File: resources/assets/uad_lists.json [hunk 6]
```diff
@@ -4103,7 +4103,7 @@
   },
   "com.samsung.android.easysetup": {
     "list": "Oem",
-    "description": "Samsung Connect Easy Setup (now SmartThings)\nUsed to connect every Samsung device you have in your house.\nWhat's weird is that Galaxy S10 has this package (available nowhere) but S9 has com.samsung.android.oneconnect\navailable on the Playstore (https://play.google.com/store/apps/details?id=com.samsung.android.oneconnect)\n#\nNeeds \"com.samsung.android.beaconmanager\" to be useful.\n",
+    "description": "Samsung Nearby device scanning\n#\nNeeds \"com.samsung.android.beaconmanager\" to be useful.\n",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:4106]:
```suggestion
    "description": "Samsung Nearby device scanning\nRequires \"com.samsung.android.beaconmanager\" to be useful.",
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]
Nitpick, but after this change, this PR can be merged.

**Comment by AnonymousWP** [comment]
@prakharb5 Please follow up on this so we can merge this ASAP. 🙂

**Review by AnonymousWP — APPROVED** [review]

**Pull request merged**